cmake/Modules/FindONEMKL.cmake

```diff
@@ -0,0 +1,52 @@
+set(ONEMKL_FOUND FALSE)
+
+set(ONEMKL_LIBRARIES)
+
+# MKL configuration will not be imported into the build system.
+# CMPLR_ROOT and MKLROOT are the same in the Pytorch development bundle.
+if(DEFINED ENV{CMPLR_ROOT})
+  set(CMPLR_ROOT $ENV{CMPLR_ROOT})
+endif()
+
+if(NOT CMPLR_ROOT)
```

Comment:
Using `SYCL_FOUND` instead. Or create a cmake variable to record `CMPLR_ROOT` in `SYCL.cmake`.

Reply:
Changed to using `SYCL_ROOT` as `CMPLR_ROOT` when `SYCL_FOUND` is `TRUE`. The path calculated based on `SYCL_ROOT` will be checked for existence to avoid potential issues caused by explicitly specified `SYCL_ROOT`.